### PR TITLE
fix: broken property access from k8s configmap/secret via RBAC 

### PR DESCRIPTION
**Pull request by sfrevel**
fix: broken property access from k8s configmap/secret via RBAC  in gravitee-apim-rest-api-standalone-container

## Issue

https://github.com/gravitee-io/issues/issues/8095

## Description

remove duplicated/overwriting spring property handling, fix path for loading gravitee.yml in container

### CODE REVIEW

File: gravitee-apim-rest-api/gravitee-apim-rest-api-standalone/gravitee-apim-rest-api-standalone-distribution/src/main/resources/bin/gravitee [hunk 1]
```diff
@@ -28,9 +28,9 @@ fi
 
 # Searching for configuration 
 GRAVITEE_OPTS=""
-if [ -f "/etc/gravitee-management/gravitee.yml" ]
+if [ -f "/opt/graviteeio-management-api/config/gravitee.yml" ]
 then
-	GRAVITEE_OPTS="-Dgravitee.conf=/etc/gravitee-management/gravitee.yml"
+	GRAVITEE_OPTS="-Dgravitee.conf=/opt/graviteeio-management-api/config/gravitee.yml"
```
Comment (gaetanmaisse) [inline gravitee-apim-rest-api/gravitee-apim-rest-api-standalone/gravitee-apim-rest-api-standalone-distribution/src/main/resources/bin/gravitee:33]: Looks like we have to do the same here also https://github.com/gravitee-io/gravitee-api-management/blob/a88b259ade0d8373f0b188f3d7cf403365cfd08d/gravitee-apim-gateway/gravitee-apim-gateway-standalone/gravitee-apim-gateway-standalone-distribution/src/main/resources/bin/gravitee#L31

**Review by gaetanmaisse — COMMENTED** [review]
👋🏻  Hey @sfrevel, thanks for the contribution and sorry for the long delay 🙈 

Did a review but would like the input from another @gravitee-io/apim or @gravitee-io/archi member before going forward

File: gravitee-apim-rest-api/gravitee-apim-rest-api-standalone/gravitee-apim-rest-api-standalone-distribution/src/main/resources/bin/gravitee (hunk 1 above)
Comment (sfrevel) [reply gravitee-apim-rest-api/gravitee-apim-rest-api-standalone/gravitee-apim-rest-api-standalone-distribution/src/main/resources/bin/gravitee:33]: The corresponding path in the gateway pod is this one "/opt/graviteeio-gateway/config". But i didn't had any problem with this pod - maybe there the path or the GRAVITEE_OPTS is set somewhere else...

Comment (phiz71) [reply gravitee-apim-rest-api/gravitee-apim-rest-api-standalone/gravitee-apim-rest-api-standalone-distribution/src/main/resources/bin/gravitee:33]: About this, there is a default mechanism here: https://github.com/gravitee-io/gravitee-node/blob/master/gravitee-node-container/src/main/java/io/gravitee/node/container/AbstractContainer.java#L53

If the `gravitee.conf` is not set, the config file is read from `gravitee.home`.

IMHO your modification can be considered as a breaking change for some users (for instance who are running Gravitee APIM from zip files) who have a custom config file in this folder.

**Comment by sfrevel** [comment]
Is there any action planned on this by the maintainer? I can not recommend gravitee to my customer if bugs and PRs take such a long time... As far as I can see I'm not the only one which is waiting such a long time - looks like there is a general problem in the handling of contributions.

**Comment by kamiiiel** [comment]
This PR is closed and a new PR is opened to address the issue.
https://github.com/gravitee-io/gravitee-api-management/pull/2661

**Pull request closed without merging**